app/models/solidus_bolt/bolt_configuration.rb

```diff
@@ -44,6 +44,14 @@ def embed_js
       end
     end
 
+    def account_js
+      if BoltConfiguration.environments[environment].zero?
```

Comment:
If BoltConfiguration isn't setup by env variables, I get an error on line 40 (undefined `.zero?` for nil) - could you please modify this line and line 40 to avoid the error?

Reply:
I saw this issue with the `embed_js` method as well.
I'll modify them both in a separate PR to avoid issues on both